### ISSUE TITLE
Duplicate Twitter buttons

### ISSUE THREAD

**Issue by diniscorreia** [tmort/Socialite#107]
Twitter buttons are now being duplicated (actually, it looks like it creates and additional button for any other tweet button on the page).

Even [the project's home page](http://socialitejs.com/) is showing the problem (two tweet buttons appear under Demo).

Probably some change on Twitter's end?

**Comment by osvaldasvalutis** [comment]
And in some cases the buttons are tripled.

**Comment by joneslloyd** [comment]
The same story with me

**Comment by alexcasalboni** [comment]
Same problem.

Everything works fine if I remove [this line](https://github.com/tmort/Socialite/blob/master/extensions/socialite.twitter.js#L54) inside *twitterInit*, which seems to create a second anchor element. 
In fact, the whole *twitterInit* function appears to be useless now. Maybe it's just a deprecated workaround?

**Comment by joneslloyd** [comment]
Hmm, that doesn't seem to be working for me.

**Comment by alexcasalboni** [comment]
@joneslloyd sorry, I updated my comment. 

I'm fixing it on my side by removing/commenting *twitterActivate*, for now. My hypothesis is that [Twitter](http://platform.twitter.com/widgets.js) is now binding the widgets automatically, and therefore calling *window.twttr.widgets.load();* is not required anymore.

I'll be waiting for a more official fix.

**Comment by joneslloyd** [comment]
@alexcasalboni Thanks! Is that along with the whole twitterInit function removed?

I'll try this and see if it solves it :)

**Comment by alexcasalboni** [comment]
@joneslloyd no, *twitterInit* is still required otherwise the widget won't show up at all.

**Comment by joneslloyd** [comment]
I see, thank you.

That's actually not working for me. Now I'm getting the other social networks loading, and the unstyled <a href> text of the Twitter share button (without it loading into the twitter share button).

**Comment by diniscorreia** [comment]
Yep, I ended up removing ´window.twttr.widgets.load();´ to fix it.
I suspect the same, it's probably being loaded automatically. Not sure if the Twitter docs changed, but right now it only mentions ´twttr.widgets.load()´ when new content is appended to the DOM.

**Comment by tmort** [comment]
@diniscorreia Can you provide a live link to the issue on your site? Unfortunately not seeing the issue, even on the homepage. 

Here is what I am seeing: 
![2015-08-13_0710](https://cloud.githubusercontent.com/assets/809598/9248526/72d51b3a-418a-11e5-9941-186ddeaba667.png)

**Comment by alexcasalboni** [comment]
@tmort maybe you have a cached version of the Twitter JS in your browser. I can see two Twitter buttons on the homepage as well.

<img width="525" alt="socialite-bug" src="https://cloud.githubusercontent.com/assets/2457588/9248621/925caea4-41bd-11e5-916a-3b01b9444566.png">

**Comment by tmort** [comment]
@alexcasalboni Got it. Will check out when I'm in the office later today. @diniscorreia I'll give your fix a try first - if that is the solution I'll have an update ASAP. 

Thanks to all of you for creating the thread and catching this.

**Comment by alexcasalboni** [comment]
@tmort any update on this? Do you think our current hotfix might create problems? 
I mean, if Twitter decides to rollback this update, no widgets will show up at all. Can you think of any better workaround to handle both cases?

**Comment by diniscorreia** [comment]
@alexcasalboni not the best solution, but if you need a quick workaround that doesn't mess with the plugin, you can just use CSS to hide all the iframes inside that element except for the first child.

**Comment by Kedarnath-Rothe** [comment]
Please assign this issue to me
